pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -779,27 +881,25 @@ func (nc *Controller) monitorNodeHealth() error {
 			zoneToNodeConditions[utilnode.GetZoneKey(node)] = append(zoneToNodeConditions[utilnode.GetZoneKey(node)], currentReadyCondition)
 		}
 
-		nodeHealthData := nc.nodeHealthMap.getDeepCopy(node.Name)
-		if nodeHealthData == nil {
-			klog.Errorf("Skipping %v node processing: health data doesn't exist.", node.Name)
+		pods, err := nc.getPodsAssignedToNode(node.Name)
+		if err != nil {
+			utilruntime.HandleError(fmt.Errorf("Unable to list pods of node %v: %v", node.Name, err))
```

Comment:
fmt.Errorf() should start with small letter

here and everywhere else

Reply:
Done.